### ISSUE TITLE
Integration with Klassy's outlines

### ISSUE THREAD

**Issue by Kevadroz** [matinlotfali/KDE-Rounded-Corners#179]
![Captura de pantalla_20240310_212708](https://github.com/matinlotfali/KDE-Rounded-Corners/assets/16049149/5ac68d44-0efb-41ac-8a62-04b8779c4a23)
Klassy has an option to make it's outlines change color (including alpha) when you hover over a titlebar button.
I propose creating an option to fetch the color from Klassy and apply it to an outline.

**Comment by matinlotfali** [comment]
Does Klassy support soft integrations? I mean is there a way to read a variable without the need to add Klassy as a dependency of the project?

**Comment by Kevadroz** [comment]
I don't think so, but I guess It wouldn't be a hard thing to implement, so a issue or a pull request could be opened on klassy's repo to add support for this.
